package.json

```diff
@@ -165,6 +167,8 @@
     "redux-persist": "^6.0.0",
     "reselect": "^4.1.6",
     "resize-observer-polyfill": "^1.5.1",
+    "scrypt-js": "^3.0.1",
+    "semver": "^7.3.8",
```

Comment:
do we need this additional dependencies for this PR? if so, why?

Reply:
@BitHighlander ?

Reply:
from what I understand there was a hoisting issue between yarn hdwallet, and our project built in yarn3. the project does error if these packages are not specified in the package.json. not sure on the LoE needed to remove them.

Reply:
if web errors when @keepkey packages are added, and requires these, you need to add them to your packages `peerDependencies`